src/todo/todo.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type TodoDocument = Todo & Document;
+
+@Schema()
```

Comment:
add here the timestamps in order to get better information with the createAt and updatedAt fields